flake.nix

```diff
@@ -8,16 +8,19 @@
     # Custom packages
     polykey-cli.url = "github:MatrixAI/Polykey-CLI";
     polykey-cli.inputs.nixpkgs.follows = "nixpkgs"; # Inheriting the nixpkgs input above; pinned
+
+    gitignore-nix.url = "github:hercules-ci/gitignore.nix";
+    gitignore-nix.inputs.nixpkgs.follows = "nixpkgs";
   };
 
-  outputs = inputs@{ nixpkgs, flake-utils, ... }:
+  outputs = inputs@{ nixpkgs, flake-utils, gitignore-nix, ... }:
     let
       overlay = import ./overlays/overlay.nix {
-        inherit inputs; # Import the inputs into the overlay.nix
+        inherit inputs gitignore-nix; # Import the inputs and gitignore-nix into the overlay.nix
```

Comment:
Doesn't `inputs` already contain all inputs packages? This seems superfluous?

Reply:
Refer to https://github.com/MatrixAI/nixpkgs-matrix/pull/4#discussion_r1719450249

tldr: yes you're right and i've removed it